src/app/signalen/page.tsx

```diff
@@ -14,56 +14,87 @@ import {
   TextArea,
 } from '@amsterdam/design-system-react'
 import { useRouter } from 'next/navigation'
-import { FormEvent, useState } from 'react'
+import { useEffect, useState } from 'react'
+import { useForm, useWatch } from 'react-hook-form'
+import { formatErrors } from './_utils/formatErrors'
+import { addErrorCountToPageTitle } from './_utils/addErrorCountToPageTitle'
+import { FormErrorList } from './_components/FormErrorList'
+import { useFormContext } from './FormContext'
 
-function Contact() {
-  const router = useRouter()
-  const [textareaLength, setTextareaLength] = useState(0)
+function Home() {
+  const { formData, updateFormData } = useFormContext()
+
+  const {
+    control,
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm()
 
-  const handleSubmit = (e: FormEvent) => {
-    e.preventDefault()
-    router.push('/signalen/vul-aan')
+  const body = useWatch({
+    control,
+    name: 'body',
+    defaultValue: formData.body || '',
+  })
+
+  const router = useRouter()
+  const onSubmit = (data) => {
+    updateFormData(data)
+    router.push('/signalen/vul-aan-1')
   }
 
+  // Add error count to doc title
+  const formattedErrors = formatErrors(errors)
+  const [documentTitle, setDocumentTitle] = useState<string>()
+
+  useEffect(() => {
+    setDocumentTitle(document.title)
+  }, [])
+
+  useEffect(() => {
+    if (documentTitle) {
+      addErrorCountToPageTitle(formattedErrors, documentTitle)
+    }
+  }, [formattedErrors, documentTitle])
+
   return (
-    <Grid paddingBottom="medium">
-      <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
-        <form className="ams-gap--md" onSubmit={handleSubmit}>
-          <Heading>Doe een melding</Heading>
+    <Grid.Cell span={{ narrow: 4, medium: 6, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 2 }}>
+      <form className="ams-gap--md" onSubmit={handleSubmit(onSubmit)}>
```

Comment:
I think I spotted a `display: grid !important` somewhere. This looks kind of hacky. Why did you not use a `Column` component?

Reply:
`ams-gap--md` is meant as a utility class, this is the intended usage. It saves you from having to add an unnecessary `div` to your markup.